stock-tracker/src/components/compare/StockPrices.js

```diff
@@ -0,0 +1,160 @@
+  const [percentChange2, setPercentChange2] = useState(null);
+
+  const handleSymbolChange = (event, symbolNumber) => {
+    const symbol = event.target.value;
+    if (symbolNumber === 1) {
+      setSymbol1(symbol);
+    } else if (symbolNumber === 2) {
+      setSymbol2(symbol);
+    }
+  };
+
+  const fetchStockPrice = async () => {
+    if (!symbol1 || !symbol2) {
+      setErrorMessage('Please enter two valid stock symbols for comparison');
+      return;
+    }
+
+    try {
+      setErrorMessage('');
+
+      const response1 = await fetch(
+        `https://finnhub.io/api/v1/quote?symbol=${symbol1}&token=chn4661r01qsjpubcga0chn4661r01qsjpubcgag`
+      );
+      const data1 = await response1.json();
+      setCurrentPrice1(data1.c);
+      setChange1(data1.d);
+      setPercentChange1(data1.dp);
+
+      const companyResponse1 = await fetch(
+        `https://finnhub.io/api/v1/stock/profile2?symbol=${symbol1}&token=chn4661r01qsjpubcga0chn4661r01qsjpubcgag`
+      );
+      const companyData1 = await companyResponse1.json();
+      setCompanyName1(companyData1.name);
+
+      const response2 = await fetch(
+        `https://finnhub.io/api/v1/quote?symbol=${symbol2}&token=chn4661r01qsjpubcga0chn4661r01qsjpubcgag`
+      );
+      const data2 = await response2.json();
+      setCurrentPrice2(data2.c);
+      setChange2(data2.d);
+      setPercentChange2(data2.dp);
+
+      const companyResponse2 = await fetch(
+        `https://finnhub.io/api/v1/stock/profile2?symbol=${symbol2}&token=chn4661r01qsjpubcga0chn4661r01qsjpubcgag`
+      );
+      const companyData2 = await companyResponse2.json();
+      setCompanyName2(companyData2.name);
+
+      const recommendationResponse1 = await fetch(
+        `https://finnhub.io/api/v1/stock/recommendation?symbol=${symbol1}&token=chn4661r01qsjpubcga0chn4661r01qsjpubcgag`
+      );
+      const recommendationData1 = await recommendationResponse1.json();
+      setRecommendation1(recommendationData1[0]);
+
+      const recommendationResponse2 = await fetch(
+        `https://finnhub.io/api/v1/stock/recommendation?symbol=${symbol2}&token=chn4661r01qsjpubcga0chn4661r01qsjpubcgag`
+      );
+      const recommendationData2 = await recommendationResponse2.json();
+      setRecommendation2(recommendationData2[0]);
+    } catch (error) {
+      console.error('Error fetching stock price:', error);
+    }
+  };
+
+  useEffect(() => {
+    fetchStockPrice();
+  }, []);
+
+  return (
+    <div>
+      <h1>Stock Comparison</h1>
+      <div>
+        <input
+          type="text"
+          value={symbol1}
+          onChange={(event) => handleSymbolChange(event, 1)}
+          placeholder="Enter stock symbol 1"
+        />
+        <input
+          type="text"
+          value={symbol2}
+          onChange={(event) => handleSymbolChange(event, 2)}
+          placeholder="Enter stock symbol 2"
+        />
+        <button onClick={fetchStockPrice}>COMPARE</button>
+      </div>
```

Comment:
You might be able to center these elements like so with bootstrap classes by wrapping in a div:

```suggestion
      <div className="d-flex justify-content-center">
        <h1>Stock Comparison</h1>
        <div>
          <input
            type="text"
            value={symbol1}
            onChange={(event) => handleSymbolChange(event, 1)}
            placeholder="Enter stock symbol 1"
          />
          <input
            type="text"
            value={symbol2}
            onChange={(event) => handleSymbolChange(event, 2)}
            placeholder="Enter stock symbol 2"
          />
          <button onClick={fetchStockPrice}>COMPARE</button>
        </div>
      </div>
```